src/common/gateway/gateway.service.ts

```diff
@@ -94,6 +94,11 @@ export class GatewayService {
     return result;
   }
 
+  async getAccountsBulk(addresses: string[]): Promise<Account[]> {
+    const result = await this.create('address/bulk', GatewayComponentRequest.addressDetails, addresses);
```

Comment:
we should create a new component instead of using `GatewayComponentRequest.addressDetails`

Reply:
done